spec/features/merchants/items/index_spec.rb

```diff
@@ -36,26 +36,21 @@
 
       visit "/merchants/#{merch_1.id}/items"
 
-      within("#item-#{item_1.id}") do
-        expect(item_1.status).to eq("disabled")
-
-        first(:button, "Enable").click
-
-        expect(page).to have_button("Disable")
-        expect(current_path).to eq("/merchants/#{merch_1.id}/items")
-        updated_item_1 = Item.find(item_1.id)
-        expect(updated_item_1.status).to eq("enabled")
-      end
-
-      within("#item-#{item_2.id}") do
+      within("#Enabled-items") do
         expect(item_2.status).to eq("enabled")
         first(:button, "Disable").click
-
-        expect(page).to have_button("Enable")
         expect(current_path).to eq("/merchants/#{merch_1.id}/items")
         updated_item_2 = Item.find(item_2.id)
         expect(updated_item_2.status).to eq("disabled")
       end
+
+      within("#Disabled-items") do
+        expect(item_1.status).to eq("disabled")
+        first(:button, "Enable").click
+        expect(current_path).to eq("/merchants/#{merch_1.id}/items")
+        updated_item_1 = Item.find(item_1.id)
+        expect(updated_item_1.status).to eq("enabled")
+      end
     end
   end
 end
```

Comment:
Oooh I'm gonna have to take a look at my enable/disable tests too after I implement this functionality on admin merchants